src/pages/City/CreateCity/CreateCity.tsx

```diff
@@ -45,15 +47,53 @@ import{
   failUpdateAction,
 } from "../../../components/Notifications/Messages"
 import { descriptionValidation } from "../../../models/GllobalValidations/DescriptionValidation";
-
+import RegionFollower from "../../../models/Region/RegionFollower";
+import User from "../../../models/UserTable/User";
+import UserApi from "../../../api/UserApi";
+import TextArea from "antd/lib/input/TextArea";
+import CityAdmin from "../../../models/City/CityAdmin";
 
 const CreateCity = () => {
   const { id } = useParams();
   const history = useHistory();
+  const location = useLocation();
+  const followerPath = "/regions/follower/";
 
   const [loading, setLoading] = useState(false);
+  const [appealRegion, setAppealRegion] = useState<RegionProfile>(new RegionProfile());
+  const [regionFollower, setRegionFollower] = useState<RegionFollower>(new RegionFollower());
   const [city, setCity] = useState<CityProfile>(new CityProfile());
   const [regions, setRegions] = useState<RegionProfile[]>([]);
+  const [applicant, setApplicant] = useState<User>(new User());
+  const [activeUser, setActiveUser] = useState<User>(new User());
+  const [isFollowerPath, setIsFollowerPath] = useState<boolean>(location.pathname.includes(followerPath));
+
+  //const createNotification = async(userId : string, message : string) => {
+  //  await NotificationBoxApi.createNotifications(
+  //    [userId],
+  //    message + ": ",
+  //    NotificationBoxApi.NotificationTypes.UserNotifications,
+  //    `/cities/${id}`,
+  //    cityName
+  //    );
+  //}
```

Comment:
Do you need these comments?

Reply:
They will be realized in the next commit